project.clj

```diff
@@ -3,7 +3,8 @@
   :url "https://github.com/RickMoynihan/lein-tools-deps"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/tools.deps.alpha "0.3.260"]]
-
+  :dependencies [[org.clojure/clojure "1.9.0"]
```

Comment:
I added this dependency on clojure 1.9.0 because in any case this plugin doesn't work with < 1.9.0 :-)